sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2168,6 +2168,15 @@ object SQLConf {
       .checkValue(v => Set(1, 2).contains(v), "Valid versions are 1 and 2")
       .createWithDefault(2)
 
+  val STREAMING_ROCKSDB_VIRTUAL_COL_FAMILY_ENABLED =
```

Comment:
TODO: remove this config, and pass virtual col family as an argument similar as `useColumnFamilies` per operator based